### PR TITLE
hotfix/domain hosting order

### PR DESCRIPTION
**Pull request by AxelPeter**
UXMWEB-3935

### CODE REVIEW

File: client/app/hosting/hosting.service.js [hunk 1]
```diff
@@ -181,14 +181,14 @@
...
-                return this.OvhHttp.get(`/hosting/web/${serviceName}`, {
-                    rootPath: "apiv6"
-                }).then((data) => data).catch((http) => {
-                    if (_.isArray(catchOpt) && _.indexOf(catchOpt, http.status) !== -1) {
-                        return null;
-                    }
-                    return this.$q.reject(http);
-                });
+                return this.$http.get(`/hosting/web/${serviceName}`)
+                    .then((response) => response.data)
```
Comment (jleveugle) [inline client/app/hosting/hosting.service.js]: `.then(response => response.data)`

Comment (FredericEspiau) [reply client/app/hosting/hosting.service.js]: Doesn't eslint prevent us from writing this way?

Comment (AxelPeter) [reply client/app/hosting/hosting.service.js]: It's added automatically when i launch "make dev"

Comment (jleveugle) [reply client/app/hosting/hosting.service.js]: Ok, coding style rules so :p

I got used to deleting unnecessary characters xD

**Pull request merged**